src/utils/http.ts

```diff
@@ -16,8 +17,12 @@ const request = async <T>(url: string, opts?: RequestInit): Promise<T> => {
     body: JSON.stringify(opts?.body),
   });
   // TODO handle error
```

Comment:
```suggestion
```
@mvn-luanvu-hn now is it possible to delete this comment?

Reply:
@jinmayamashita I think I will create another PR to handle error in the future
In that PR I will do
- Use toast to show error of 5xx error
- Internationalize error message

Reply:
@mvn-luanvu-hn 
Sure. Can you add TODOs such as toast issue to the comments in the code?

Reply:
```suggestion
  // TODO handle error
  // - Use toast to show error of 5xx error
  // - Internationalize error message
```
@mvn-luanvu-hn I agree. :+1: How about this?

Reply:
@jinmayamashita Yes, I agree with you. I just committed your suggestion.